addons/core/CfgSounds.hpp

```diff
@@ -2,14 +2,14 @@ class CfgSounds
 {
     class TFAR_rotatorPush
     {
-        name = "TFAR - Rotator Switch (Push)";
+        name = "$STR_TFAR_Rotator_Push";
```

Comment:
These names are never seen by anyone. No need to have them in a stringtable

Reply:
ok